packages/SwingSet/src/vats/network/router.js

```diff
@@ -36,7 +36,7 @@ export default function makeRouter(sep = '/') {
   const prefixToRoute = makeStore('prefix');
   return harden({
     getRoutes(addr) {
-      const parts = addr.split('/');
+      const parts = addr.split(sep);
```

Comment:
note: the flexibility this implies might not be realistic, when we need to match the behavior of an external system. compatibility is a bear :)